.changelog

```diff
@@ -13,3 +13,4 @@
 - Add Redfish to power management protocols
 - Specifies Port 22 is required for Server (bsc#1177975)
 - Added procedure for adding virt guests to Client Cfg
+- General updates to the Client Cfg Guide
```

Comment:
```suggestion
- Updated GPG keys section of the Client Cfg Guide
```

Reply:
Oh, I was being sneaky and including all the updates I did in PR #653 as well ;)